src/app/shared/components/table/enrolment-list-filter/enrolment-list-filter.component.scss

```diff
@@ -0,0 +1,7 @@
+.filter-button {
+  margin-right: 10px;
+  background-color: gray;
```

Comment:
I didn't have rgbs or anything specific for this, but when I do I guess I should add this to scss variables @dawidgil

Reply:
They are in theme files. I refactored it to use css variables as it make more sense (for me) to have it this way, because we are supporting 2 different styles for 2 applications (Switchboard and Stedin)

Reply:
@dawidgil yeah, I just am not sure if the color I'm supposed to use is in there, because there's no RGBs on the mocks atm. I will just use the --button-disabled from theme and can change later if it doesn't look right!